### PR TITLE
4 mxnet mlp

### PR DESCRIPTION
**Pull request by mushi251**
I have added the python scripts as well as updated the notebooks to also include validation metric evaluation through each epoch (I noticed the val accuracy is greater than the traning accuracy in most of these epochs. Is this correct or is something wrong?)

### CODE REVIEW

**Review by trevor-yu-087 — APPROVED** [review]

File: multi-layer-perceptron/mxnet_test.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
+'''
+Wat.ai Python/Julia framework benchmark comparison Project
+Milestone 1
```
Comment (UrbanPistek) [inline multi-layer-perceptron/mxnet_test.py:3]: Small but I like this top level comment, copied it to my script, just gives a nice overview of what is going on.

**Review by UrbanPistek — APPROVED** [review]
My only main comment for better clarity would be to break out stuff into seperate functions like `load_data`, `init_mlp`, `train` for example to make the structure and what is happening a bit more clear.

Also, group varibles like `learning rate, epochs, batch_size` etc together instead of just scattering them about.
